networking/change-block-size.md

```diff
@@ -0,0 +1,303 @@
+
+### Concepts
+
+#### Best practices
+
+- **Create IP pools within the same cluster CIDR** 
+
+   Kubernetes expects that all pods have IP addresses within the same cluster CIDR. Although {{site.prodname}} technically supports using multiple disjoint IP pool CIDRs, we do not recommend it; pods allocated addresses outside of the Kubernetes cluster CIDR will lose network connectivity. 
+  
+- **Determine IP pool block size before installing {{site.prodname}}** 
+
+   Because the `blockSize` field cannot be edited directly after installation, you must use a three-step process after installation to change `blockSize` to mimimize disruptions to pod connectivity.
+
+### Before you begin...
+
+**Required**
+
+- Verify that you are using {{site.prodname}} IPAM. This guide is only relevant if you are using Calico IPAM.
+
+  ssh to one of your Kubernetes nodes and view the CNI configuration.  
+
+    ```bash
+    cat /etc/cni/net.d/10-calico.conflist
+     ```
+  Look for the "type" entry:
+  <pre>
+     "ipam": {
+           "type": "calico-ipam"
+      }, 
+  </pre>
+
+  If the type is “calico-ipam”, you are good to go. If the IPAM is set to something else, or the 10-calico.conflist file does not exist, you cannot use this feature in your cluster. 
+
+- Verify orchestrator support for changing the pod network CIDR
+
+  Although Kubernetes supports changing the pod network CIDR, not all orchestrators do. For example, **OpenShift** does not support this feature as described in [`osm_cluster_network_cidr configuration`](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-cluster-variables). Check your orchestrator documentation to verify. 
+
+**Recommended**
+
+Understand the basics of [Calico IPAM]({{site.baseurl}}/networking/get-started-ip-addresses)
+
+### How to
+
+>**Important!** Make sure that you follow the steps in the order described to minimize pod connectivity disruption. Pods may lose connectivity when they are redeployed, and may lose external connectivity while in the temporary pool. Also, when pods are deleted, applications may be temporarily unavailable (depending on the type of application). Plan accordingly. 
+{: .alert .alert-danger }
+
+#### Change the IP pool block size
+
+By default, the {{site.prodname}} IPAM block size for an IP pool is /26. To expand from the default size /26, lower the `blockSize` (for example, /8). To shrink the `blockSize` from the default /26, raise the number (for example, /28). 
+
+The high-level steps to follow are:
+
+1. Add a new temporary IP pool.  
+**Note**: The new IP pool must not be within the same cluster CIDR.
+1. Disable the old IP pool.  
+**Note**: Disabling an IP pool prevents only new IP address allocations; it does not affect etworking of existing pods.
+1. Delete pods from the old IP pool.  
+This includes any new pods that may have been created with the old IP pool prior to disabling the pool. Verify that new pods get an address from the new IP pool.
```

Comment:
I think we can get rid of line 76 - don't think it adds any additional information.